packages/core/package.json

```
@@ -30,7 +30,7 @@
     "clean:plugin": "expo-module clean plugin",
     "circularDepCheck": "madge --circular dist/js/index.js && madge --circular metro.js && madge --circular expo.js",
     "test": "yarn test:sdk && yarn test:tools",
-    "test:sdk": "npx jest",
+    "test:sdk": "npx jest --maxWorkers=8",
```

Change has no effect, lets limit it only on local devices.
```suggestion
    "test:sdk": "sh -c 'if [ \"$CI\" = \"true\" ]; then yarn test:sdk-ci; else yarn test:sdk-local; fi'",
    "test:sdk-ci": "npx jest",
    "test:sdk-local": "jest --maxWorkers=8",
```